app/lib/features/chat_ng/widgets/chat_bubble.dart

```diff
@@ -6,148 +6,180 @@ import 'package:flutter/material.dart';
           color: theme.colorScheme.onPrimary,
         ),
-        child: child,
+        child: bubbleContentWidget,
       ),
+      isMe: true,
     );
   }
 
   @override
   Widget build(BuildContext context) {
-    final chatTheme = Theme.of(context).chatTheme;
     final size = MediaQuery.sizeOf(context);
     final msgWidth = messageWidth?.toDouble();
     final defaultWidth =
         context.isLargeScreen ? size.width * 0.5 : size.width * 0.75;
 
-    return Container(
-      margin: const EdgeInsets.symmetric(horizontal: 8),
-      child: Row(
-        mainAxisAlignment: bubbleAlignment,
-        crossAxisAlignment: CrossAxisAlignment.end,
-        children: [
-          const SizedBox(width: 5),
-          IntrinsicWidth(
-            child: Container(
-              constraints: BoxConstraints(maxWidth: msgWidth ?? defaultWidth),
-              decoration: decoration,
-              child: Padding(
-                padding: const EdgeInsets.symmetric(
-                  horizontal: 16,
-                  vertical: 16,
-                ),
-                child: Column(
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  mainAxisSize: MainAxisSize.min,
-                  children: [
-                    child,
-                    const SizedBox(height: 8),
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.end,
-                      children: [
-                        if (isEdited)
-                          Text(
-                            L10n.of(context).edited,
-                            style: chatTheme.emptyChatPlaceholderTextStyle
-                                .copyWith(fontSize: 12),
-                          ),
-                        if (isEdited && timestamp != null)
-                          Padding(
-                            padding: const EdgeInsets.symmetric(horizontal: 2),
-                            child: Text(
-                              '.',
-                              style: chatTheme.emptyChatPlaceholderTextStyle
-                                  .copyWith(fontSize: 12),
-                            ),
-                          ),
-                        if (timestamp != null)
-                          MessageTimestampWidget(
-                            timestamp: timestamp.expect('should not be null'),
-                          ),
-                      ],
-                    ),
-                  ],
+    return IntrinsicWidth(
+      child: Container(
+        margin: const EdgeInsets.symmetric(horizontal: 14),
+        padding: const EdgeInsets.all(10),
+        constraints: BoxConstraints(maxWidth: msgWidth ?? defaultWidth),
+        decoration: decoration,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            if (displayName != null) ...[
+              Text(
+                displayName!,
```

Comment:
not a fan of `!` at all, that being refactored later has bitten us in the path and if it happens is super hard to figure out where, so I _really_ don't like seeing it, especially in rendering code. You probably have it here because there is a lint triggered if you kept it out, but we can easily get around that via re-assignment:

```suggestion
    final name = displayName;
    
    return IntrinsicWidth(
      child: Container(
        margin: const EdgeInsets.symmetric(horizontal: 14),
        padding: const EdgeInsets.all(10),
        constraints: BoxConstraints(maxWidth: msgWidth ?? defaultWidth),
        decoration: decoration,
        child: Column(
          crossAxisAlignment: CrossAxisAlignment.start,
          mainAxisSize: MainAxisSize.min,
          children: [
            if (name != null) ...[
              Text(
                name,
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2941/commits/1173c59806fb599e44ebdb3519c9cc59d8e61022